libpod/boltdb_state_internal.go

```diff
@@ -497,6 +497,16 @@ func (s *BoltState) getVolumeFromDB(name []byte, volume *Volume, volBkt *bolt.Bu
 		}
 	}
 
+	// Retrieve volume driver
+	if volume.UsesVolumeDriver() {
+		plugin, err := s.runtime.getVolumePlugin(volume.config.Driver)
+		if err != nil {
+			logrus.Errorf("Volume %s uses volume plugin %s, but it cannot be accessed - some functionality may not be available: %v", volume.Name(), volume.config.Driver, err)
```

Comment:
I wonder why errors are non-fatal.  Does Docker behave the same way?

Could you add a comment?

Reply:
This is all to ensure that, if the plugin goes away, Podman does not become nonfunctional - we can still remove the volume from the plugin.

Reply:
Well, from Podman, but not from the Plugin.

Reply:
Done